### PR TITLE
DSPDC-915 Tweaks for ENCODE S3->GCS deployment

### PR DESCRIPTION
**Pull request by danxmoran**
First, an unrelated change: I split the ClinVar and ENCODE workflows into separate deployments, so we can work on each independently.

Second, a related change: I updated the AWS account module to accept a policy description. AWS is odd (in a cool way) and users must be granted access to use APIs at all, even for things like reading from public S3 buckets. I updated our ENCODE template to grant `GetObjects` permission to the `encode-public` bucket for the users we generate.

### CODE REVIEW

File: templates/helm/orchestration-workflows/clinvar/templates/workflow.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
   resetValues: true
   chart:
     git: git://github.com/databiosphere/clinvar-ingest.git
-    ref: master
+    ref: ra-DSPDC-1009-no-rows
```
Comment (danxmoran) [inline templates/helm/orchestration-workflows/clinvar/templates/workflow.yaml:38]: TODO: Revert back to `master`

File: templates/helm/orchestration-workflows/encode/templates/encode.yaml [hunk 2]
```diff
@@ -35,7 +45,7 @@ spec:
   resetValues: true
   chart:
     git: git://github.com/databiosphere/encode-ingest.git
-    ref: master
+    ref: dm-port-example-DSPDC-915
```
Comment (danxmoran) [inline templates/helm/orchestration-workflows/encode/templates/encode.yaml:48]: TODO: Revert back to `master`

File: templates/helm/orchestration-workflows/clinvar/templates/workflow.yaml (hunk 1 above)
Comment (raaidbroad) [reply templates/helm/orchestration-workflows/clinvar/templates/workflow.yaml:38]: should be good to revert now, is merged!

File: templates/helm/orchestration-workflows/clinvar/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "env": { "type": "string", "enum": ["dev"] },
+    "properties": {
+      "enable": { "type": "boolean" },
+      "schedule": { "type": "string" }
+    }
+  },
+  "required": ["env", "enable"]
```
Comment (raaidbroad) [inline templates/helm/orchestration-workflows/clinvar/values.schema.json:11]: `enable` isn't in the right scope to be required here, I think. Shouldn't `enable` and `schedule` be pulled up to be on the same level as `env`? If not, they're not correctly nested as sub-properties under the `env` property.

File: templates/helm/orchestration-workflows/encode/values.schema.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "env": { "type": "string", "enum": ["dev"] },
+    "properties": {
+      "enable": { "type": "boolean" },
+      "schedule": { "type": "string" }
+    }
+  },
+  "required": ["env", "enable"]
```
Comment (raaidbroad) [inline templates/helm/orchestration-workflows/encode/values.schema.json:11]: `enable` isn't in the right scope to be required here, I think. Shouldn't `enable` and `schedule` be pulled up to be on the same level as `env`? If not, they're not correctly nested as sub-properties under the `env` property.

**Pull request merged**